flow360/component/simulation/entity_info.py

```diff
@@ -133,6 +134,14 @@ class GeometryEntityInfo(EntityInfoModel):
 
     global_bounding_box: Optional[BoundingBoxType] = pd.Field(None)
 
+    # pylint: disable=no-member
+    default_geometry_accuracy: Optional[LengthType.Positive] = pd.Field(
+        None,
+        description="The smallest length scale that will be resolved accurately by the surface meshing process. "
```

Comment:
no need for documentation.

```suggestion
        description="The default value based on uploaded geometry for geometry_accuracy"
```